llvm/test/CodeGen/AMDGPU/machine-cse-copyprop.mir

```diff
@@ -0,0 +1,22 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+# RUN: llc -mtriple=amdgcn -run-pass=machine-cse -verify-machineinstrs -o - %s | FileCheck %s
+
+# Test to ensure that this does not crash on undefs
+---
+name: machine-cse-copyprop
```

Comment:
Name should indicate it's testing the undef case

Reply:
done